CHANGELOG.rst

```diff
@@ -1,10 +1,11 @@
 CHANGELOG
 =========
 
-2.9.4.rc1 (2016-05-16)
-----------------------
+3.0.0 (2016-05-26)
+------------------
 
-* Adds support for the django CMS's ability to better handle plugin creation
+* Adds django CMS 3.3 support
+* Backwards incompatible with django CMS < 3.3
```

Comment:
This isn't really a changelog entry, but should definitely be part of the readme (which you have done).